ps_facetedsearch.php

```diff
@@ -1503,4 +1513,88 @@ public function getWidgetVariables($hookName, array $configuration)
     {
         return [];
     }
+
+    /**
+     * Install review indexes table
+     */
+    public function rebuildCommentIndexTable()
+    {
+        $this->getDatabase()->execute('DROP TABLE IF EXISTS `' . _DB_PREFIX_ . 'layered_comment_index`');
+
+        $this->getDatabase()->execute('DROP TABLE IF EXISTS `' . _DB_PREFIX_ . 'layered_comment_index_log`');
+
+        $this->getDatabase()->execute(
+            'CREATE TABLE `' . _DB_PREFIX_ . 'layered_comment_index` (
+            `id_product` INT  NOT NULL,
+            `score` INT NOT NULL,
+            `avg_score` FLOAT NOT NULL,
```

Comment:
Do we really need to use Float like that maybe we can do 
```suggestion
            `avg_score` FLOAT(5, 4) NOT NULL,
```
Meaning we can do 9.9999
No need to do 25.390843984 ^^

Reply:
correct :D